### DISCUSSION TITLE
Update for Safari?

### DISCUSSION THREAD

**Discussion by FirleFrantz** [darkreader/darkreader#12536] [General]
I wonder if there will be an update for safari at some point. Chrome, Firefox and co are at v4. is that what they are planning?

**Comment by gwcali** [comment]
Could definitely use an updated version for Safari and keep it running smoothly. Things have been cropping up where Dark Reader needs to be disabled for websites to render properly.

**Reply by gwcali** [reply]
@alexanderby

**Comment by bigplayer-ai** [comment]
I hope someone makes dark reader open source on safari 
Because it feels very slow to update and zero maintenance…
